server/ngwmn/services/ngwmn.py

```diff
@@ -309,3 +310,168 @@ def get_features(latitude, longitude, service_root=SERVICE_ROOT):
         raise ServiceException()
 
     return response.json()
+
+
+def get_statistic(agency_cd, site_no, stat_type, service_root=SERVICE_ROOT_CACHE):
+    # base_url = "http://cida-eros-ngwmndev:8080/ngwmn_cache/direct/json/"
+    # base_url = "https://cida.usgs.gov/ngwmn_cache/direct/json/"
+    stats_url = '/'.join([service_root, 'ngwmn_cache', 'direct', 'json', stat_type, agency_cd, site_no])
+    resp = r.get(stats_url)
+
+    if resp.status_code == 404:
+        return {
+            'IS_RANKED': 'N',
+            'IS_FETCHED': 'N'
+        }
+
+    if resp.status_code != 200:
+        msg = '%s statistics fetch error from %s (reason: %s)'
+        app.logger.error(msg, resp.status_code, resp.url, resp.reason)
+        raise ServiceException()
+
+    json_txt = resp.text
+    stats = json.loads(json_txt)
+    # TODO log in trace mode
+    # print(json.dumps(stats, indent=2) + '\n')
+
+    stats['IS_FETCHED'] = 'Y'
+
+    return stats
```

Comment:
I'm thinking this function should coerce the service-returned data into the appropriate native Python type - eg, date or datetime objects, booleans (for "N" / "Y"), etc.

Reply:
I agree, I would normally use python true and false. I chose to conform the Y/N coming from NGWMN for internal consistency.

Reply:
Ah oh, I see that function isn't used directly by the UI portion.